index.html1

```diff
@@ -0,0 +1,84 @@
+alt="Scanmarker">
+
+<h2>Product Description</h2>
+<p>With a processing speed 30 times faster than manual typing, the Scanmarker will let you scan textbooks, notebooks, invoices and magazines, so students, lawyers, researchers or anyone with learning difficulties can read, study or work confidently. pricing info, customer testimonials.</p>
+<!--https://www.scanningpens.com/Data-Capture-Products-SPUS/scanmarker.html#:~:text=With%20a%20processing%20speed%2030,read%2C%20study%20or%20work%20confidently.-->
+
+<h2>Pricing Info</h2>
+<p>The Scanmarker prices range from $149-$249 depending on where you decide to purchase it from</p>
+
+<h2>Customer Reviews</h2>
+<p>"I have found the Scanmarker Air to be an excellent tool for capturing quotes for research projects and for course preparation. The software interfaces very well with Microsoft Word and with little practice it is easy to get the right motion for accurately capturing a line of text" -Paul R</p>
+
+<p>"The scanner transmits notes from a printed document/book in a fraction of the time it takes to write or type the note. Has several cool functions of translations, including text to speech. Great aid to studying large amounts." -Samriti M</p>
+
+<section>
+<h3>Submit a Customer Review Here:</h3>
+<form action="/server-page-handling-form-data" method="past">
+    <ul>
+        <li>
+            <label for="fname">First Name:</label>
+            <input type="text" id="fname" name="fname" placeholder="Input First Name Here">
+        </li>
+        <li>
+            <label for="lname">Last Name:</label>
+            <input type="text" id="lname" name="fname" placeholder="Input Last Name Here">
+        </li>
+        <li>
+            <label for="email">Email:</label>
+            <input type="email" id="email" name="email" placeholder="scan@marker.com">
+        </li>
+        <li>
+            <label for="message">Review:</label>
+            <textarea id="message" rows="5"></textarea>
+        </li>
+        <li><button type="submit">Submit</button></li>
+    </ul>
+</form>
+</section>
```

Comment:
Yay! Great job with your section here - just need to do this around your other headings!